src/main/java/com/bgsoftware/superiorskyblock/listeners/PlayersListener.java

```diff
@@ -509,7 +509,7 @@ public void onPlayerCommand(PlayerCommandPreprocessEvent e){
         Island island = plugin.getGrid().getIslandAt(e.getPlayer().getLocation());
 
         if(island != null && !island.isSpawn() && island.isVisitor(superiorPlayer, false) &&
-                plugin.getSettings().blockedVisitorsCommands.stream().anyMatch(cmd -> e.getMessage().contains(cmd))){
+                plugin.getSettings().blockedVisitorsCommands.stream().anyMatch(cmd -> e.getMessage().toLowerCase().contains(cmd))){
```

Comment:
This can be optimized by caching the result of e.getMessage().toLowerCase() in a separate variable, and then check if this variable contains the cmd in it, instead of running e.getMessage().toLowerCase() every check.

Reply:
It's right, i will do that sunday